crates/context/src/cfg.rs

```diff
@@ -125,15 +125,19 @@ impl<SPEC> CfgEnv<SPEC> {
     }
 
     /// Sets the blob target and max count over hardforks.
-    pub fn with_blob_max_and_target_count(mut self, blob_params: Vec<(SpecId, u64, u64)>) -> Self {
-        self.set_blob_max_and_target_count(blob_params);
+    pub fn with_blob_max_and_target_count(mut self, blob_taget: u64, blob_max_count: u64) -> Self {
+        self.set_blob_max_and_target_count(blob_taget, blob_max_count);
         self
     }
 
     /// Sets the blob target and max count over hardforks.
-    pub fn set_blob_max_and_target_count(&mut self, mut blob_params: Vec<(SpecId, u64, u64)>) {
-        blob_params.sort_by_key(|(id, _, _)| *id);
-        self.blob_target_and_max_count = blob_params;
+    pub fn set_blob_max_and_target_count(&mut self, blob_target: u64, blob_max_count: u64) {
+        self.blob_target_and_max_count = Some((blob_target, blob_max_count));
+    }
+
+    /// Clears the blob target and max count over hardforks.
+    pub fn clear_blob_max_and_target_count(&mut self) {
+        self.blob_target_and_max_count = None;
     }
```

Comment:
these makes sense, since this is breaking this should highlight all locations where we need to update this in reth

Reply:
Evm uses only blob_max_count field, and that is to only check if tx has less than max, so making this check optional is fine. And if num of blobs are checked outside of revm there is no need to use `set_blob_max_count` fn